scripts/build.js

```diff
@@ -11,7 +11,7 @@ var projects = [
 ];
 
 var fs = require('fs');
-let dataPath = "data";
+let dataPath = `${process.env.TOOL_DATA_DIR}/public_html/data`;
```

Comment:
I missed this at the time, but `TOOL_DATA_DIR` is normally undefined when running the update script locally. 9d842c6d66ec7c84041b4a2e4731afd9f02986ac makes this environment variable optional.